cmake/linux/toolchain-loongarch64.cmake

```diff
@@ -0,0 +1,21 @@
+# See linux/toolchain-x86_64.cmake for details about multiple load of toolchain file.
+include_guard(GLOBAL)
+
+set (CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
+
+set (CMAKE_SYSTEM_NAME "Linux")
+set (CMAKE_SYSTEM_PROCESSOR "loongarch64")
+set (CMAKE_C_COMPILER_TARGET "loongarch64-linux-gnu")
+set (CMAKE_CXX_COMPILER_TARGET "loongarch64-linux-gnu")
+set (CMAKE_ASM_COMPILER_TARGET "loongarch64-linux-gnu")
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mcmodel=extreme")
```

Comment:
Please add a comment, otherwise future readers will have no idea why `-mcmodel=extreme` is needed for loongarch64.

Reply:
OK, I'll add a comment explaining why -mcmodel=extreme is needed for loongarch64.